packages/posthog/src/definitions.ts

```diff
@@ -1,3 +1,5 @@
+import type { JsonType } from 'posthog-js';
```

Comment:
Let's define the type ourselves for now, so that you don't have to install the posthog js sdk if your app is only available on Android and iOS. Perhaps we will change this later.

Just add this to the `definitions.ts` file and import it here:

```ts
export type JsonType = string | number | boolean | null | {
    [key: string]: JsonType;
} | Array<JsonType>;
```